### DISCUSSION TITLE
Microsoft OneNote Style Boards of Notes

### DISCUSSION THREAD

**Discussion by protean-onion** [TriliumNext/Trilium#3992] [Ideas]
Hello,

Although Trilium is perfect in a lot of ways for me, I like something about OneNote a lot: you create zettelkasten style notes, so your note looks like a whiteboard with various note parts. I think it'd be great if Trilium, the mother of all note-taking apps, also had it, and enriched her garden with a richer variety of beings.

Thanks for your time.

**Comment by meichthys** [comment]
@adityaverma415 You can get the 'whiteboard' style of note using the "canvas" note type. If you're looking for the ability to add links to internal notes from within a canvas note, see #2960 and #3902

**Reply by protean-onion** [reply]
Thanks!
